fresque/default_config.py

```diff
@@ -21,7 +21,7 @@
 # secret key used to generate unique csrf token
 SECRET_KEY = 'Change-me-Im-famous'
 
-## If the authentication method is `fas`:
+# If the authentication method is `fas`:
 # To log in, the user must be a member of one of these groups
 REQUIRED_GROUPS = ('packager', 'provenpackager')
```

Comment:
Note: I don't think we'll need provenpackager, but we will need a way to know if someone is a sponsor and that might require a change either in Ipsilon or in flask-fas-openid.